### PR TITLE
Create checkDataForSubject.py

### PR DESCRIPTION
**Pull request by suhlrich**
This scripts checks that all videos are there for a given subject across multiple sessions.

Have used this for digital health day and it works nicely.

### CODE REVIEW

File: utils.py [hunk 1]
```diff
@@ -1283,4 +1283,60 @@ def checkResourceUsage():
...
+    return resourceUsage
+
+# %% Some functions for loading subject data
+
+def getSubjectNumber(subjectName):
+    subjects = requests.get(API_URL + "subjects/",
+                           headers = {"Authorization": "Token {}".format(API_TOKEN)}).json()
+    sNum = [s['id'] for s in subjects if s['name'] == subjectName]
+    if len(sNum)>1:
+        print(len(sNum) + ' subjects with the name ' + subjectName + '.')
```
Comment (antoinefalisse) [inline utils.py:1295]: Would add a warning that you will be be selecting the first one.

File: Examples/checkDataForSubject.py [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+across however many sessions are associated with them.
+
+Note: this script does not run without updating the subject name and trialnames
+for your own data.
+
+
+You will need login credentials from app.opencap.ai to run this script. You 
+can quickly create credentials from the home page: https://app.opencap.ai. 
+We recommend first running the Examples/createAuthenticationEnvFile.py script
+prior to running this script. Otherwise, you will need to to login every time
```
Comment (antoinefalisse) [inline Examples/checkDataForSubject.py:31]: typo: to to

**Review by antoinefalisse — APPROVED** [review]
@suhlrich looks good, I did not test but trust you did. I wonder if Examples in the right folder. This code is very specific to an application. Thoughts? Feel free to merge

**Comment by suhlrich** [comment]
yeah it is a bit specific, but the Examples aren't too burdensome at this point. we can reorganize later if useful.

**Pull request merged**